go/common/enclave.go

```diff
@@ -110,20 +110,13 @@ type Enclave interface {
 
 // BlockSubmissionResponse is the response sent from the enclave back to the node after ingesting a block
 type BlockSubmissionResponse struct {
-	BlockHeader             *types.Header             // The header of the consumed block.
```

Comment:
The block submission response doesn't need to return the block, as its the host who submitted the block in the first place.